### PR TITLE
Upgrade to vue2

### PR DESCRIPTION
**Pull request by freekmurze**

### CODE REVIEW

File: resources/assets/js/components/current-time.vue [hunk 1]
```diff
@@ -1,19 +1,20 @@
+<template>
```
Comment (freekmurze) [inline resources/assets/js/components/current-time.vue]: So back to vueify? Why the change of heart?

**Comment by sebastiandedeyne** [comment]
Almost, just need to review the changes regarding filters, might want to do another little refactor there.
